### ISSUE TITLE
Explore hover states based on Working Group feedback

### ISSUE THREAD

**Issue by hannalaakso** [alphagov/govuk-frontend#1853]
## What
We received some useful feedback from the Working Group for [making hover styles clearer](https://github.com/alphagov/govuk-design-system/pull/1274) and will do some exploration to answer their questions. 

This work addresses https://github.com/alphagov/govuk-frontend/issues/1417.

(We might wish to split this card into smaller ones and have an epic instead but chatted to @christopherthomasdesign who thought that for now a bigger card works.)

[Link to WG feedback summary when it's posted]

## Why
New hover styles could be a breaking change for some users so we want to use the feedback from the WG to ensure that the hover styles work for different edge cases.

## Who needs to know about this
Chris, developers

## Done when
- [ ] We have considered how the WG feedback should be reflected in the hover styles and have created some mock styles, if appropriate

We have tested how the hover styles work for:
- [ ] Text blocks eg. gov.uk figures
<img width="200" alt="text blocks with figures on gov.uk" src="https://user-images.githubusercontent.com/5007934/86380428-f4adf400-bc83-11ea-9890-24750d064629.png">

- [ ] Text blocks with icons eg. Bank Holiday page links
<img width="250" alt="text block links on gov.uk bank holiday page" src="https://user-images.githubusercontent.com/5007934/86380659-435b8e00-bc84-11ea-8c60-83ef19adde3d.png">

- [ ] Images as links
- [ ] White backgrounds and non-white backgrounds (already done but we should check this if we change anything)
- [ ] Other cases - add here 

As indicated to the WG, we also have post-review actions to:
- [ ] Consider [touch highlights](https://github.com/alphagov/govuk-frontend/issues/1205)
- [ ] Test if colour for the underline can be inherited from the text colour
- [ ] Test in different browsers/devices

We should also:
- [ ] Decide if we want to create a proposal for the changes, and create a new card if appropriate
- [ ] Decide if we want to do a pre-release of the work, and create a new card if appropriate

**Comment by kellylee-gds** [comment]
Duplicate of #1417 - will move this info to that card
